building/src/spire/src/seq.py

```diff
@@ -39,7 +40,9 @@ def sequence_supervisor(ops: setup.Operations) -> None:
     ops.add_operation("pre-deploy dns-addon", deploy.launch_dns_addon)
     ops.add_operation("pre-deploy flannel-monitor", deploy.launch_flannel_monitor)
     ops.add_operation("pre-deploy dns-monitor", deploy.launch_dns_monitor)
-    ops.add_operation("pre-deploy user-grant", deploy.launch_user_grant)
+
+    if config.user_grant_domain != '':
+        ops.add_operation("pre-deploy user-grant", deploy.launch_user_grant)
```

Comment:
We probably want to find a way to include this in the tests at some point, but yeah, good enough for now.